README.md

```diff
@@ -14,15 +14,21 @@ Turn complex GraphQL queries into optimized database queries.
 pip install graphql-compiler
 ```
 
+## Quick Overview 
+
+The GraphQL compiler turns read-only queries written in GraphQL syntax to different 
+query languages. Through the GraphQL compiler, users can write powerful queries that uncover 
+deep relationships in the data while not having to worry about the underlying database query 
+language.
+
+Furthermore, the GraphQL compiler validates queries through the use of a GraphQLSchema object 
+that specifies the underlying schema of the database. We can currently autogenerate the 
+GraphQLSchema by querying an OrientDB database, (see  [Querying OrientDB with GraphQL](#querying-orientdb-with-graphql)). In the nearby future, we plan to add 
+schema autogeneration from SQLAlchemy metadata. 
+
 For a more detailed overview and getting started guide, please see
 [our blog post](https://blog.kensho.com/compiled-graphql-as-a-database-query-language-72e106844282).
 
-To pretty-print GraphQL queries, use the included pretty-printer:
```

Comment:
Why remove this? I believe this is the only mention of this capability of the tool, so if we remove this mention without adding it elsewhere, users will be entirely unable to find it.

Reply:
I don't think that the feature is important enough to have at the top of the Github repo. Also, it's a bit vague also what it means to pretty-print a GraphQL query. I am also not sure why this one of the first things I would like to learn about the compiler.